tests/Unit/Core/Form/IdentifiableObject/CommandBuilder/Product/AbstractProductCommandBuilderTest.php

```diff
@@ -39,22 +39,13 @@ abstract class AbstractProductCommandBuilderTest extends TestCase
 {
     public const SHOP_ID = 1;
 
-    /**
-     * @var ShopConstraint
-     */
-    protected $singleShopConstraint;
+    protected const MODIFY_ALL_NAME_PREFIX = 'modify_all_shops_';
```

Comment:
Since this const is now in the abstract I guess you need to clean all the other test classes to remove the now useless const no? and use this one instead